streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -887,6 +899,7 @@ public void run() {
         boolean cleanRun = false;
         try {
             taskManager.init();
+            initLatencyWindowsIfNeeded(System.currentTimeMillis());
```

Comment:
We should never call `System.curentTimeMillis()`directly, but use `time.milliseconds()` instead.